src/Sylius/Bundle/ShopBundle/Resources/views/layout.html.twig

```diff
@@ -42,7 +42,7 @@
 
                 {{ sonata_block_render_event('sylius.shop.layout.after_header') }}
 
-                {{ render(url('sylius_shop_partial_taxon_index_by_code', {'code': 'category', 'template': '@SyliusShop/Taxon/_horizontalMenu.html.twig'})) }}
+                {{ render(url('sylius_shop_partial_channel_menu_taxon_index', {'template': '@SyliusShop/Taxon/_horizontalMenu.html.twig'})) }}
```

Comment:
I wonder whether we can use `render(controller())` to avoid the need for adding a new route.

Reply:
Yes. We can do something like this:

```
{{ render(controller(
    'sylius.controller.taxon:indexAction',
    {'_sylius': {'template': '@SyliusShop/Taxon/_horizontalMenu.html.twig', 'repository': {'method': 'findChildrenByChannelMenuTaxon', 'arguments': [sylius.channel.menuTaxon, app.request.locale]}}}
)) }}
```

I just wonder is it a better solution or not (better readability and not too much of logic in the template vs unneeded route) 🤔